tests/end_to_end/helpers/db.py

```diff
@@ -33,6 +33,7 @@ def run_query_mysql(query, host, port, user, password, database):
                          user=user,
                          password=password,
                          database=database,
+                         charset='utf8mb4',
```

Comment:
just wondering, is there any specific reason of adding this?

Reply:
Sure, if you look at the changes I've made to `test_target_postgres/snowflake.py`, you can see that the query entails inserting some special characters, if I don't specify a utf8 charset, there is an encoding error.

Reply:
yu can see inhttps://github.com/transferwise/pipelinewise/blob/fa24440ea3cbdf0baff57a80b764cfe92213010e/tests/db/tap_mysql_data.sql#L32 that the table's DDL sets charset to utf8.

Reply:
utf8mb4 is the better cousin of utf8 ;)

Reply:
Let me know if I should do it differently.

Reply:
I was only wondering why it didn't fail earlier without specifying charset in the `run_query_mysql`. But the default charset on the table level is probably the answer as you said.

Thanks you